llvm/lib/Transforms/Instrumentation/Instrumentation.cpp

```diff
@@ -32,7 +32,7 @@ class DiagnosticInfoInstrumentation : public DiagnosticInfo {
 public:
   DiagnosticInfoInstrumentation(const Twine &DiagMsg,
                                 DiagnosticSeverity Severity = DS_Warning)
-      : DiagnosticInfo(DK_Linker, Severity), Msg(DiagMsg) {}
+      : DiagnosticInfo(DK_OptimizationRemarkMissed, Severity), Msg(DiagMsg) {}
```

Comment:
The doc for enum says we need a new enum for each subcluss
Maybe add a new one, e.g DK_Instrumentation